Mogakco/Sources/Data/DataMapping/UpdateStudyIDRequestDTO.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  UpdateStudyIDRequestDTO.swift
+//  Mogakco
+//
+//  Created by 신소민 on 2022/11/26.
+//  Copyright © 2022 Mogakco. All rights reserved.
+//
+
+import Foundation
+
+struct UpdateStudyIDRequestDTO: Encodable {
+    private let chatRoomIDs: ArrayValue<StringValue>
```

Comment:
```suggestion

struct UpdateStudyIDsRequestDTO: Encodable {
    private let chatRoomIDs: ArrayValue<StringValue>
```
사소한 부분이지만 API 이름과 통일성있게 복수형이면 좋을 것 같아요!

Reply:
네이밍 지적 좋습니다. 👍 리뷰 반영했습니다.